src/simulator/instruction.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg_attr(not(test), allow(dead_code))]
+
+extern crate foxbox_taxonomy;
+
+use foxbox_taxonomy::util::Id;
+use foxbox_taxonomy::values::*;
+use foxbox_taxonomy::devices::*;
+use foxbox_taxonomy::selector::*;
+use foxbox_taxonomy::api::{WatchEvent, WatchOptions, Error};
+
+#[derive(Serialize, Deserialize, Debug)]
+/// Instructions given to the simulator.
+pub enum Instruction {
```

Comment:
This needs a few more words, to mention that, in practice, this is how clients are going to script the simulator.